pom.xml

```diff
@@ -142,6 +142,11 @@
       <version>5.11.2</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-web</artifactId>
```

Comment:
yeah, my idea was the same - just add this dependency. But maybe there is a smaller lib with the same functionality?